### ISSUE TITLE
Add a section on automated machine learning

### ISSUE THREAD

**Issue by rhiever** [amueller/scipy-2016-sklearn#77]
Perhaps the workshop already has too much material for the usual runtime (~8 hours), but how about a practical section on automated machine learning? I taught this section at the end of a version of this sklearn workshop and it was quite well-received, and I think it's a good topic to mention to beginners (even if just a teaser).

**Comment by amueller** [comment]
We are mostly focusing on things that can be applied directly in practice. Do you have an application of automl that can be run immediately and will be widely useful? Is there a two-liner in tpot that can be useful for many? I don't think auto-sklearn can be easily installed yet.

**Comment by rhiever** [comment]
Yep, check [this notebook](https://github.com/rhiever/webstep-sklearn-quickstart/blob/6b9312a3b77583262ee6379630a52555c1d3bbac/notebooks/23%20Automated%20machine%20learning.ipynb). It's a very quick coding demo of TPOT for classification and regression, working on similar data sets to the ones in the rest of the workshop.

We've gone out of our way to make TPOT very easy to install and use, even for Python newbies.

I can flesh that notebook out more if there's interest in adding AutoML to the workshop. For now I just accompany that notebook with [this talk](https://github.com/rhiever/webstep-sklearn-quickstart/blob/8f9f5861ed8c733af6ceb069134dd45942b91653/slides/automl-tpot.pptx), which takes about 20-30 mins to deliver.
